LoginSimulator/Program.cs

```diff
@@ -0,0 +1,56 @@
+﻿using System;
+
+namespace LoginSimulator
+{
+  internal static class Program
+  {
+    private static void Main(string[] args)
+    {
+      TestCredentials();
+    }
+
+    private static void TestCredentials()
+    {
+      const string storedUserName = "tEsT";
+      const string storedUserPassword = "1234";
+      
+      const string successfulLoginMessage = "Logged in!";
+      
+      var userName = PromptString("Username: ");
+      var userPassword = PromptString("Password: ");
+
+      var isValidCredential = string.Equals(userName, storedUserName, StringComparison.OrdinalIgnoreCase) &&
+                              string.Equals(userPassword, storedUserPassword);
+      if (isValidCredential)
+      {
+        Console.WriteLine(successfulLoginMessage);
+      }
+    }
+    
+    private static string PromptString(string message, bool isKeyVisible = false)
+    {
+      if (string.IsNullOrWhiteSpace(message))
```

Comment:
In general, don't do null checks for internal methods.  You will drive yourself crazy.